recipes/dulwich/meta.yaml

```diff
@@ -0,0 +1,97 @@
+  # noarch: python
+  number: 1
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    # When setuptools is available add the `--single-version-externally-managed --record record.txt` above.
+    - setuptools
+    # if your project compiles code (such as a C extension) then add `toolchain` as a build requirement.
+    - toolchain
+  run:
+    - python
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - dulwich
+    - dulwich.tests
+    - dulwich.archive
+    - dulwich.object_store
+    - dulwich.objects
+    - dulwich.tests.utils
+    - dulwich.repo
+    - dulwich.client
+    - dulwich.config
+    - dulwich.protocol
+    - dulwich.pack
+    - dulwich.diff_tree
+    - dulwich.index
+    - dulwich.file
+    - dulwich.errors
+    - dulwich.hooks
+    - dulwich.ignore
+    - dulwich.objectspec
+    - dulwich.patch
+    - dulwich.reflog
+    - dulwich.refs
+    - dulwich.server
+    - dulwich.walk
+    - dulwich.web
+
+about:
+  home: https://www.dulwich.io/
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGLP.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: Apache
+  # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
+  license_family: Apache
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See http://conda.pydata.org/docs/building/meta-yaml.html#license-file
+  license_file: CONTRIBUTING.md
+  summary: 'Python Git Library'
+
+  # The remaining entries in this section are optional, but recommended
+  description: |
+    Python implementation of the Git file formats and protocols, without the need to have git installed.
+    All functionality is available in pure Python. Optional C extensions can be built for improved performance.
+    The project is named after the part of London that Mr. and Mrs. Git live in in the particular Monty Python sketch.
+  doc_url: https://www.dulwich.io/docs/
+  dev_url: https://github.com/jelmer/dulwich
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
+    - mikofski
+    - jelmer
```

Comment:
@jelmer -- are you ok being added as a co-maintainer of the `dulwhich` recipe on conda-forge?

https://conda-forge.org/docs/recipe.html#maintainer-role